Formula/wireguard-tools.rb

```diff
@@ -14,8 +13,13 @@ class WireguardTools < Formula
     sha256 "bc4cebbd91c650870f7ddb34f1f7957597e6c91d997aee772c4e958238be0546" => :el_capitan
   end
 
+  depends_on "bash"
```

Comment:
Does this really depend on bash 4 being installed? At run-time?

Reply:
Yes, it really does. It won't run on bash 3, as it uses a number of newer additions to the shell language.